js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeRequestedPartitionsLink.tsx

```diff
@@ -63,15 +66,22 @@ export const AutomaterializeRequestedPartitionsLink = ({runIds, partitionKeys}:
     }
 
     return runIds ? (
-      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} />
+      <PartitionAndRunList runIds={runIds} partitionKeys={filteredPartitionKeys} intent={intent} />
     ) : (
       <VirtualizedPartitionList partitionKeys={partitionKeys} />
     );
   };
 
   return (
     <>
-      <ButtonLink onClick={() => setIsOpen(true)}>{label}</ButtonLink>
+      <Box>
+        <Tag intent={intent}>{label}</Tag>
+        <ButtonLink onClick={() => setIsOpen(true)}>
+          <small style={{marginLeft: 10}}>
```

Comment:
Same comment as on the other PR -- `small` isn't a widely used element, and I'm not even sure we have default styles in place for it. 😄 You can use our `<Caption>` component when you need small text.

Additionally, instead of applying inline `marginLeft` style on this element, you can use a `gap` value on the `Box` container.

```
<Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
  <Tag ... />
  <ButtonLink>...</ButtonLink>
</Box>
```